Foodhut/src/sections/introduction.html

```diff
@@ -19,12 +19,14 @@ <h1 class="header-introduction">
       Best cooks and best delivery guys all at your service. Hot tasty food will reach you in 60 minutes.
     </p>
     <form class="search-box-container flex between">
-      <div class="search-black-icon">
-        <img src="./assets/images/search-icon-black.svg" alt="search black icon"/>
+      <div>
```

Comment:
Should not use element without styling

Reply:
I used background-image to add image instead <img> elements

Reply:
@CBinh710 Please consider to read those article before you use background img instead of image
https://sbx.webflow.io/image-or-background-image
https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image